papers/ring_vrf/pedersen_vrf.tex

```diff
@@ -99,52 +112,75 @@ \section{Ring VRF construction}% {Ring VRFs from the Pedersen VRF}
+%sigma protocol that shows equality of the discrete logarithm of $\PreOut$
+%in base $\In$ and the committed secret key in $\compk$.
+%COMMENT Handan: We should either explain the following statement more or remove it
+%We nevertheless define security only for our ring VRF constructions,
+% as \PedVRF itself exhibits surprising properties.
 % \footnote{We do not define security for \PedVRF because pseudo-randomness becomes too interesting}
 
 \smallskip
 % \subsection{Pedersen rVRF-AD}
 
-As described in \S\ref{sec:overview},
-we instantiate a rVRF-AD from \PedVRF plus a ring commitment scheme
- $\rVRF.\{ \CommitRing, \OpenRing \}$.
-In our construction, $\rVRF.\Eval = \PedVRF.\Eval$.
-and $\rVRF.\KeyGen = \PedVRF.\KeyGen$.
-We alter $\rVRF.\KeyGen$ somewhat in \S\ref{subsec:rvrf_faster}
-by adopting a SNARK friendly public key,
- although the \pk here still exists under the hood.
-We discuss further nuances of this in \S\ref{subsec:AML_KYC}.
-
-As mentioned in \S\ref{sec:overview}, the signing algorithm of our ring VRF construction shows that the public key of the committed secret key is in the ring. Therefore, we need a zero-knowledge ring membership proof for the relation \Lring
-which handles both $\PedVRF.\OpenKey$ and $\rVRF.\OpenKey$ efficiently.
-% \vspace{-0.1in}
-$$ \Lring = \Setst{ \compk, \comring }{
-    \eprint{ \exists \openpk,\openring \textrm{\ s.t.\ } }{}
-    \genfrac{}{}{0pt}{}{\PedVRF.\OpenKey(\compk,\openpk) \quad}{\,\, = \rVRF.\OpenRing(\comring,\openring)}
-} \mathperiod $$
-
-
-\def\tmpaux{\aux \doubleplus \compk \doubleplus \piring \doubleplus \comring}
-\def\tmpeprintaux{\eprint{\aux'}{\tmpaux}}
-\def\tmpindent{\hspace*{5pt}}
+Now, we are ready to introduce our ring VRF construction  $ \rVRF $.
+
 \begin{itemize}
-\item $\rVRF.\rSign\eprint{ : }{}(\sk,\openring,\msg,\aux)\eprint{ \mapsto \rho}{}$
-	returns $\rho = (\compk,\piring,\sigma)$ \eprint{where \\}{\\ where} 
-	\tmpindent $(\openpk,\compk) \leftarrow \PedVRF.\CommitKey$,  \\
-	\tmpindent $\piring \leftarrow \NIZK_{\Lring}.\Prove((\compk,\comring); (\openpk,\openring))$, \\
-	\tmpindent $\aux' \leftarrow \tmpaux$,  \\
-	\tmpindent $\sigma \leftarrow \PedVRF.\Sign(\sk,\openpk,\msg, \aux')$.
-\item $\rVRF.\rVerify : (\comring,\msg,\aux,\rho) \mapsto \Out \,\, \lor \perp$ \,
-	parses $\rho$ as $(\compk,\piring,\sigma)$, \eprint{next }{}sets $\aux' \leftarrow \tmpaux$,
-	aborts if $\NIZK_{\Lring}.\Verify((\compk,\comring); \piring)$ fails,
-	and returns $\PedVRF.\Verify(\compk,\msg, \aux', \sigma)$.
+	\item $\rVRF.\KeyGen $ is the same as $ \PedVRF.\KeyGen$.
+	\item $\rVRF.\Eval $ is the same as $ \PedVRF.\Eval$.
+	\item $ \rVRF.\CommitRing: \pkring \rightarrow \comring$ given a set of public keys $ \pkring $, it returns a Merkle tree root where the leaves are elements of  $ \pkring$. 
+	
+	We note that our construction works fine by replacing  $ \comring $ by $ \pkring $. However, the asymptotic complexity of our protocol with $ \comring $ is much more better
```

Comment:
I would say: "We note that our construction preserves its security properties when replacing $\comring$ by $\pkring$. However, by using $\comring$ we clearly improve the efficiency of our protocol." Is this what you had in mind?